src/test/java/org/apache/sling/resourceresolver/impl/mapping/MapEntriesTest.java

```diff
@@ -161,8 +161,8 @@ public void setup() throws Exception {
 
         Optional<ResourceResolverMetrics> metrics = Optional.empty();
 
-        mapEntries = Mockito.spy(new MapEntries(resourceResolverFactory, bundleContext, eventAdmin, stringInterpolationProvider, metrics));
-        final Field aliasMapField = MapEntries.class.getDeclaredField("aliasMap");
+        mapEntries = /*Mockito.spy(*/new MapEntries(resourceResolverFactory, bundleContext, eventAdmin, stringInterpolationProvider, metrics);//);
```

Comment:
@jsedding Do you have any idea why the spy was used here? I commented it to ease debugging and not having it didn't lead to any test failures.

Reply:
Nope. No idea.